demo/web/src/main/notebooks/B. Data science, ML, & AI/B2 NumPy and Deephaven.md

```diff
@@ -80,14 +80,13 @@ print(f"b flattened:\n{b_flattened}")
 Let's integrate NumPy with Deephaven by creating a table with some data.  That data will be an independent variable (X) and sum of sine waves of different amplitudes and frequencies (Y).  We can construct this noisy signal with NumPy's built in function for a sine wave.
 
 ```python
-from deephaven.TableTools import emptyTable
+from deephaven2 import empty_table
 
 def generate_noisy_signal(x):
     return 3.5 * np.sin(x) + 1.5 * np.sin(2.5 * x) + 0.75 * np.sin(3.5 * x) + np.random.normal()
 
-data_table = emptyTable(1000).update(
-    "X = 0.01 * i",
-    "Y = (double)generate_noisy_signal(X)"
+data_table = empty_table(size = 1000).update(
+    formulas = ["X = 0.01 * i", "Y = (double)generate_noisy_signal(X)"]
```

Comment:
Do we want to use the named parameters even when the parameter is required? I'm fine with either way and even using both styles, but if we care about being consistent then we should follow one of them for consistency (I know I didn't use `formulas=["XYZ"]` in my updates)